sw/blas/gemm/src/gemm.h

```diff
@@ -69,55 +76,126 @@ void sc_st_gemm(precision_t prec, uint32_t expand, uint32_t setup_ssr,
 
         switch (prec) {
             case FP64:
-                if (baseline) {
-                    gemm_fp64_naive(frac_m, n, k, (double*)a + offsetA,
-                                    lda_strided, transa, (double*)b, ldb,
-                                    transb, (double*)c + offsetC, ldc_strided,
-                                    (double)beta);
-                } else {
-                    gemm_fp64_opt(frac_m, n, k, (double*)a + offsetA,
-                                  lda_strided, transa, (double*)b, ldb, transb,
-                                  (double*)c + offsetC, ldc_strided, &beta,
-                                  setup_ssr);
+                switch (impl) {
+                    case NAIVE:
+                        gemm_fp64_naive(frac_m, n, k, (double*)a + offsetA,
+                                        lda_strided, transa, (double*)b, ldb,
+                                        transb, (double*)c + offsetC,
+                                        ldc_strided, (double)beta);
+                        break;
+                    case NAIVE_UNROLLED:
+                        printf(
+                            "Naive unrolled implementation not supported for "
```

Comment:
Move this into validation cfg

Reply:
I would add that this can be done also for all other unsupported configurations, we can remove them from `gemm.h` to keep it concise, and move it to datagen.